test/WebJobs.Script.Tests/Rpc/GenericWorkerProviderTests.cs

```diff
@@ -18,47 +18,45 @@ namespace Microsoft.Azure.WebJobs.Script.Tests.Rpc
     public class GenericWorkerProviderTests
     {
         private static string rootPath = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+        private static string testLanguagePath = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
```

Comment:
how are these test directories cleaned up? I ask because I've seen other tests littering my hard drive with TONs of temp dirs that aren't deleted :)

Reply:
there is a [finally](https://github.com/Azure/azure-functions-host/blob/ef311c8c4a46e6494e0b6b4af359586bb1295667/test/WebJobs.Script.Tests/Rpc/GenericWorkerProviderTests.cs#L132) block that deletes the test directory.